python/lsst/ap/association/association.py

```diff
@@ -107,27 +107,34 @@ def run(self, dia_sources, exposure):
 
         dia_objects = self.level1_db.load_dia_objects(expMd)
 
-        association_result = self.associate_sources(dia_objects, dia_sources)
-        updated_obj_ids = association_result.updated_obj_ids
+        updated_obj_ids = self.associate_sources(dia_objects, dia_sources)
 
         # Store newly associated DIASources.
         self.level1_db.store_dia_sources(dia_sources, updated_obj_ids)
+        self.level1_db._db_cursor.execute("select * from dia_sources")
```

Comment:
This line has no effect, as far as I can tell.

Reply:
Correct, this was a debug line I accidentally left in.